gbpservice/nfp/configurator/drivers/firewall/asav/asav_fw_driver.py

```diff
@@ -276,6 +312,14 @@ def configure_interfaces(self, context, resource_data):
             LOG.error(msg)
             raise Exception(msg)
 
+        try:
+            status = self._log_forwarding(mgmt_ip)
```

Comment:
We need to check unsuccessful error status and handle error just like what is done for exception.